support-lambdas/bigquery-acquisitions-publisher/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "bigquery-acquisitions-publisher",
+	"version": "1.0.0",
+	"main": "index.js",
+	"license": "MIT",
+	"scripts": {
+		"test": "vitest --run",
+		"test-watch": "vitest",
+		"build": "esbuild --bundle --platform=node --target=node20 --outfile=target/typescript/index.js typescript/index.ts"
```

Comment:
We're using webpack in support-frontend and I'm just using TSC in the support-workers work, so it could be good to standardise at some point, but I think this is fine for this PR.

Reply:
Ah that's interesting. We started out with just `tsc` but ran into an issue which I can't now remember, maybe Paul can? I suggested `esbuild` as I think it's worked well in support-service-lambdas.

Reply:
Yeah I think it has, I don't have particularly strong opinions either way but consistency is always good. Let's leave as is for now though and we can think about it at a later date.